OutOfSchool/OutOfSchool.WebApi/Controllers/OrganizationController.cs

```diff
@@ -33,8 +35,9 @@ public OrganizationController(ILogger<OrganizationController> logger, IOrganizat
         /// Get all organization from the database.
         /// </summary>
         /// <returns>List of all organizations.</returns>
+        [ProducesResponseType(StatusCodes.Status200OK)]
```

Comment:
could it produce responses other than 200?

Reply:
Do you mean 204 No Content?

Reply:
could it produce 400 or 500 response?